src/content/blog/2024-06-20-simple-open-payments-guide.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: ../../layouts/BlogLayout.astro
+title: "A Simple Guide to the Open Payments Standard"
+description: Learn how the Open Payments standard makes online payments easier and more accessible for everyone.
+date: 2024-06-20
+slug: simple-open-payments-guide
+author: Sarah Jones
+author_url: https://www.linkedin.com/in/sarah-jones-ba6bb6b9
+tags:
+  - Interledger
+  - Open Payments
+---
+
+## The Current Digital Payments Landscape
+
+Handling payments is a crucial part of many online applications. Whether you’re running an eCommerce site selling products, a fundraising platform accepting donations, a streaming service charging for content, or a subscription service requiring monthly fees, digital payments are at the core of the operation.
+
+Many application developers rely on third-party payment gateways to handle these transactions. Companies like PayPal, Stripe, and Square offer services to process credit card payments on behalf of the application. In return for these services, applications incur various fees, which can include monthly fees, flat rates per transaction, or a percentage of the transaction amounts. This approach, while convenient, introduces additional expenses and makes the application reliant on a third-party provider. This reliance can limit control over the user experience, such as forcing users to trust third-party providers with sensitive information like credit card details.
+
+An alternative is for applications to integrate directly with their bank's payment processing services. This method can offer lower transaction fees and increase control over the payment process. However, it requires significant development effort and is not always possible. Additionally, switching to a different bank becomes very challenging due to the extensive integration work that was already completed.
+
+This problem gets even more complicated when we consider situations where either the sender or the recipient of funds does not have a bank account. What if a payment has to take place between a bank and a mobile money provider? Now the application would have to also integrate with the mobile money provider. Custom integration becomes an expansive problem.
```

Comment:
![custom](https://github.com/interledger/interledger.org-developers/assets/1461498/85f7a1e2-afc4-4db2-9785-1c7e1206fed9)

Reply:
I think "receiving bank" can be removed here, given the paragraph talks about moving money between a sending bank _directly to_ a mobile money provider

Reply:
@mkurapov my idea here was that the fundraising app did a direct integration with their bank (where they'd like to receive funds). Now the sender has a bank, that's fine because bank A can probably talk to bank B. But if the sender has a mobile money account or digital wallet we may have issues.

Reply:
@JoblersTune makes sense, in that case, maybe we can just rename from "receiving" to "merchant" bank maybe (or "fundraisers" bank), and then "customer" bank? To remove any confusion from sending/receiving